opendistro-elasticsearch-alerting-kibana.release-notes.md

```diff
@@ -1,4 +1,9 @@
-## Version 0.9.0.0 (Current)
+## Version 0.10.0.0 (Current)
```

Comment:
Isn't the current one 1.0 ?

Reply:
This is for Open Distro version 0.10.0.0 not 1.0.0.0. 

OD 0.10 -> ES 6.8
OD 1.0 -> ES 7.0.1